packages/svelteui-core/src/components/Input/Input.svelte

```diff
@@ -87,12 +88,7 @@ Base component to create custom inputs
 
 <Box {...wrapperProps} class={getStyles({ css: override })} {...$$restProps}>
 	{#if icon}
-		<svelte:component
-			this={icon}
-			size={iconProps.size}
-			color={iconProps.color}
-			class={classes.icon}
-		/>
+		<IconRenderer {icon} className={classes.icon} {...iconProps} iconSize={16} />
```

Comment:
There is some weird misalignment going with the icon, but I think that problem existed before, will have to fix in another PR (just leaving here to remind me, please ignore me)